plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/ReadSessionCreator.java

```diff
@@ -34,26 +37,31 @@
 import static com.google.cloud.bigquery.TableDefinition.Type.VIEW;
 import static io.trino.spi.StandardErrorCode.NOT_SUPPORTED;
 import static java.lang.String.format;
+import static java.time.temporal.ChronoUnit.MILLIS;
 import static java.util.stream.Collectors.toList;
 
 // A helper class, also handles view materialization
 public class ReadSessionCreator
 {
+    private static final Logger log = Logger.get(ReadSessionCreator.class);
```

Comment:
nit: Add an empty line between `Logger log` and `BigQueryClientFactory bigQueryClientFactory`.

Reply:
fixed